jq/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM debian:bullseye-slim
+
+LABEL MAINTAINER="oss@sentry.io"
+
+RUN set -x \
```

Comment:
wasn't sure what this did until reading this:
https://opensource.com/article/20/5/optimize-container-builds

Can we link that as a comment?

Reply:
or another link if you used another site

Reply:
or is this common for dockerfiles and I just didn't know?
```
DEBIAN_FRONTEND=noninteractive
```
skips interactive prompt during installation
```
rm -rf /var/lib/apt/lists/*
```
Clear package list so it isn't stale

Reply:
Yeah I do this whenever I use apt in Docker, to reduce size as much as possible, and I've seen it in other containers as well, e.g. https://github.com/docker-library/python/blob/230bb8c07e599a1e230b63895647e677d109c5c0/3.10/slim-bullseye/Dockerfile#L17